src/main/java/com/example/waytogo/route/service/api/RouteService.java

```diff
@@ -0,0 +1,16 @@
+package com.example.waytogo.route.service.api;
+
+import com.example.waytogo.route.model.entity.Route;
+
+import java.util.Optional;
+import java.util.UUID;
+
+public interface RouteService {
+
+    Optional<Route> getRouteById(UUID id);
+    Route save(Route route);
+
+    void updateRouteById(UUID routeId, Route route);
+
+    void delete(Route route);
+}
```

Comment:
o widzisz nawet sam jestes swiadomomy ze to niz nie zwraca to ja nw jaki optional przy delete wtedy dales